index.html

```diff
@@ -0,0 +1,353 @@
+<!doctype html>
+<html lang="en-us">
+<head>
+  <title>Media Capture and Streams Extensions</title>
+  <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
+  <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
+  <script class='remove'>
+  "use strict";
+  // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+  var respecConfig = {
+    group: "webrtc",
+    xref: ["html", "infra", "permissions", "dom", "mediacapture-streams"],
+    edDraftURI: "https://w3c.github.io/mediacapture-extensions/",
+    editors:  [
+      {name: "Jan-Ivar Bruaroey", company: "Mozilla Corporation", w3cid: 79152},
+    ],
+    shortName: "mediacapture-extensions",
+    specStatus: "ED",
```

Comment:
From editor's meeting: Make this an unofficial draft for now
```suggestion
    specStatus: "UD",
```